gtda/local_homology/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""The module :mod:`gtda.local` implements transformers to generate
+local persistence diagrams."""
+
+# The LocalVietorisBase was ignored, since it is not to be used.
```

Comment:
Nice comment!  But it's OK to remove it, it is expected that some classes will not be exposed to the user.